help/destinations/catalog/email-marketing/oracle-eloqua-api.md

```diff
@@ -0,0 +1,165 @@
+---
+keywords: crm;CRM;crm destinations;api oracle eloqua;api oracle eloqua crm destination
+title: (API) Oracle Eloqua connection
+description: The (API) Oracle Eloqua destination allows you to export your account data and activate it within Oracle Eloqua for your business needs.
+exl-id: 
```

Comment:
Removed however please check the template still has the exl-id.

Reply:
Ah, I see what happened here. The link you provide above is the Got location for the published example page on Experience League. Since this is a live page, it will have an `exl-id`. 

For the latest template, please always go to this link (https://experienceleague.adobe.com/docs/experience-platform/destinations/destination-sdk/document-destination/documentation-instructions.html?lang=en#documentation-template) and select **Download the template here** to get the latest version of the template, which does not have an `exl-id`.